packages/spruce-skill/interface/pages/_app.js

```diff
@@ -54,6 +58,21 @@ export default class MyApp extends App {
 
 		return (
 			<Container>
+				{pageProps.initialProps.legacyStylesheet && (
+					<Head>
+						<meta
```

Comment:
Delete this since you also have it in document.js

Reply:
In the next docs it shows having the meta tag twice with the same key to avoid duplications https://nextjs.org/docs#populating-head